docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -31,32 +30,80 @@ Some use cases for the Shipping Provider API are:
 - Third-party logistics
 - Create a combination of in-store pickup and shipping options for shoppers
 
+This article is a guide to developing an app that will make your shipping rates available to merchants and shoppers on demand.
+
 ### Prerequisites
 
 * Required [OAuth](/api-docs/getting-started/authentication#authentication_oauth-scopes) scopes: `Information and Settings`
 * BigCommerce [single-click app](https://developer.bigcommerce.com/api-docs/apps/guide/types#single-click)
 
-## Sign-up
+## Shipping app overview
 
-When creating your app, add the shipping data to our Carrier Registry, so it returns to shoppers on the front-end.
+<!--
+    title: ### Carrier Service API
 
-To get your app setup, send an email to
-<a href="mailto:shippingproviderapi@bigcommerce.com">shippingproviderapi@bigcommerce.com</a>.
+    data: //s3.amazonaws.com/user-content.stoplight.io/6012/1552664566834
+-->
 
-Please include:
+### Carrier Service API
 
-- Name
-- Partner ID
-- App ID (see below)
-- Email
-- Description of the app
-- [Your service URLs](#your-service-urls) 
+![### Carrier Service API
+](//s3.amazonaws.com/user-content.stoplight.io/6012/1552664566834 "### Carrier Service API
+")
+
+### Single-carrier vs multi-carrier apps
+
+Only one registered shipping carrier can be associated with an app. This [registered carrier](#definitions) can provide quotes from multiple downstream carriers. 
+
+BigCommerce makes a distinction between single-carrier and multi-carrier shipping providers, but because only one registered shipping carrier can be associated with an app, the primary difference that this distinction makes is in how the quote displays in the customer's cart at checkout.  If your app is registered as a single carrier, the name of the carrier providing the quote will appear beside the name of the shipping quote in the customer's list of shipping rate options. The carrier name will not appear in quotes from multi-carrier apps. The following images illustrate the difference:

```

Comment:
Run-on sentence

Reply:
technically all those clauses are in proper relationships with each other, but I take your point that this is wordy.  revised.